components/channel_header/index.js

```diff
@@ -79,7 +78,7 @@ function makeMapStateToProps() {
             rhsState: getRhsState(state),
             rhsOpen: getIsRhsOpen(state),
             isFavorite: isCurrentChannelFavorite(state),
-            isReadOnly: isCurrentChannelReadOnly(state),
+            isReadOnly: false,
```

Comment:
Could you file a follow up ticket to remove these props entirely and clean up the related code in the components? I don't think that needs to be done for 6.0, but I'd like to avoid leaving potentially a bunch of unused code around.

Reply:
> Could you file a follow up ticket to remove these props entirely and clean up the related code in the components? I don't think that needs to be done for 6.0, but I'd like to avoid leaving potentially a bunch of unused code around.

Totally agree. I had mentioned it in [another comment](https://github.com/mattermost/mattermost-webapp/pull/8541#discussion_r692342155), but I created [a ticket](https://mattermost.atlassian.net/browse/MM-38043) to remove these. I started removing them in this PR but it turned into a cascading set of changes that are better done without a time constraint.